### PR TITLE
chore(logs & metrics): remove debug statements, add metrics

### PR DESCRIPTION
**Pull request by emjburns**
removing some debug statements that print a lot and instead creating some more metrics so we can alert when things don't happen.

### CODE REVIEW

File: keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/CheckScheduler.kt [hunk 1]
```diff
@@ -96,16 +100,12 @@ class CheckScheduler(
         }
       }
       runBlocking { job.join() }
-      log.debug("Scheduled resource validation complete")
```
Comment (robfletcher) [inline keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/CheckScheduler.kt:99]: The value (such as it is) of these logs is in just knowing the loops didn't get stuck. We'll be able to see that from telemetry, right?

**Review by robfletcher — APPROVED** [review]
I like this. It is good.

**Review by gal-yardeni — APPROVED** [review]
LGTM

File: keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/CheckScheduler.kt (hunk 1 above)
Comment (emjburns) [reply keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/CheckScheduler.kt:99]: Yep, I added a duration metric, so if we see that increasing we will know that something is funky. Or at least, that is my theory.

**Review by luispollo — APPROVED** [review]
Good cleanup!

While you're at it, I found the other message that is always confusing to me, if you wanna change that. It's this line: https://github.com/spinnaker/keel/blob/66d7fac0a1a6901c6165f1ccf1f3d3c6074ec326/keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt#L113

To me, that always reads backwards: if the check is stale, we should check again, not skip checks, but that's obviously not what this is doing, so maybe we can just improve the message?

**Pull request merged**